backup_and_restore/application_backup_and_restore/oadp-use-cases/oadp-usecase-legacy-aws-plugin.adoc

```diff
@@ -0,0 +1,12 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="oadp-using-legacy-aws-plugin"]
+= Using the legacy-aws Velero plugin
+include::_attributes/common-attributes.adoc[]
+:context: oadp-using-legacy-aws-plugin
+
+toc::[]
+
+If you are using an {aws-short} S3-compatible backup storage location, you might get a `SignatureDoesNotMatch` error while backing up your application. This is because, some backup storage locations still use the `AWS SDK for Go V1` instead of the newer `AWS SDK for Go V2`. The S3 APIs in older version of the GO SDK cause errors. To solve this issue, you can use the `legacy-aws` Velero plugin in the `DataProtectionApplication` custom resource (CR). The `legacy-aws` Velero plugin uses the older `AWS SDK for Go V1` to interact with {aws-short} S3.
```

Comment:
I think this is more curate:
If you are using an {aws-short} S3-compatible backup storage location, you might encounter a SignatureDoesNotMatch error while backing up your application. This error occurs because some backup storage locations still use older versions of the S3 APIs, which are incompatible with the newer AWS SDK for Go V2. To resolve this issue, you can use the legacy-aws Velero plugin in the DataProtectionApplication custom resource (CR). The legacy-aws Velero plugin relies on the older AWS SDK for Go V1, which is compatible with these legacy S3 APIs, ensuring successful backups.

Reply:
Thank you Amos. I will work on these changes.